### PR TITLE
US4 Kate

### PR DESCRIPTION
**Pull request by katebetsko**
done User Story #4

### CODE REVIEW

**Review by MinaKhilwati — APPROVED** [review]
Well done!

File: configuration.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 browser=chrome
-url=https://qa1.vytrack.com
+url=https://qa.transmuda.com/
 driver_username=user1
```
Comment (SulaimanHayat36) [inline configuration.properties:3]: Thank you. I was wondering why it was a different url

File: pom.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
-            <version>4.9.1</version>
+            <version>4.12.1</version>
```
Comment (SulaimanHayat36) [inline pom.xml]: Thank you again. This confused me for more than an hour

**Review by SulaimanHayat36 — APPROVED** [review]
Good Job. Everything looks good

**Pull request merged**